Makefile

```diff
@@ -0,0 +1,13 @@
+install:
+	npm install
+
+dev--linux: install
+	multitail -s 2 \
+		-cT ANSI -t "[playground] " -l "npm start --prefix packages/playground" \
+		-cT ANSI -t "[slate-renderer] " -l "npm run build:watch --prefix packages/slate-renderer" \
+		-cT ANSI -t "[docx-cleaner] " -l "npm run build:watch --prefix packages/docx-cleaner" \
+		-cT ANSI -t "[slate-commons] " -l "npm run build:watch --prefix packages/slate-commons" \
+		-cT ANSI -t "[slate-editor] " -l "npm run build:watch --prefix packages/slate-editor" \
+		-cT ANSI -t "[slate-hyperscript] " -l "npm run build:watch --prefix packages/slate-hyperscript" \
+		-cT ANSI -t "[slate-lists] " -l "npm run build:watch --prefix packages/slate-lists" \
+		-cT ANSI -t "[slate-types] " -l "npm run build:watch --prefix packages/slate-types";
```

Comment:
I think we should use lerna here, e.g.

```
lerna run build:watch --scope @prezly/slate-types
```

Reply:
How is this better?
For this to work you would need to have lerna installed globally. Otherwise we would need to specify each watch script in root package.json.